### ISSUE TITLE
Alpine / musl build broken for 2.4: unknown type name 'cpu_set_t'

### ISSUE THREAD

**Issue by TimWolla** [haproxy/haproxy#1235]
## Code Report

Tool: gcc

```
   CC      src/cpuset.o
In file included from include/haproxy/global-t.h:28,
                 from include/haproxy/global.h:26,
                 from include/haproxy/fd.h:33,
                 from src/ev_epoll.c:19:
include/haproxy/cpuset-t.h:17:22: error: unknown type name 'cpu_set_t'
   17 | # define CPUSET_REPR cpu_set_t
      |                      ^~~~~~~~~
include/haproxy/cpuset-t.h:37:2: note: in expansion of macro 'CPUSET_REPR'
   37 |  CPUSET_REPR cpuset;
      |  ^~~~~~~~~~~
```

see: https://github.com/docker-library/haproxy/runs/2421829708?check_suite_focus=true

## Output of `haproxy -vv` and `uname -a`

GitHub Actions.

**Comment by a-denoyelle** [comment]
Thanks for the report I will look at it

**Comment by sodabrew** [comment]
Yep, this is affecting the 2.4-dev17 builds on Alpine. I'll test out with the Debian images for now, but hope that dev18 will solve it next week!

**Comment by sodabrew** [comment]
I think this ticket can be closed since 2.4-dev18 was released with a fix.

**Comment by TimWolla** [comment]
> I think this ticket can be closed since 2.4-dev18 was released with a fix.

Indeed, thanks.
